start.sh

```diff
@@ -26,6 +28,42 @@ start_ethminer() {
     ethminer_pid=$!
 }
 
+start_livepeer() {
+    echo "Starting livepeer in the background..."
+    livepeer $livepeer_args &
+    livepeer_pid=$!
+}
+
+stop_livepeer() {
```

Comment:
Since `livepeer` is now started in the background I think we should call this function in `cleanup()` so that when the bash script exits the `livepeer` process is also killed along with the `ethminer` process.